### PR TITLE
Add review files

### PR DESCRIPTION
**Pull request by Obote**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (Obote) [inline index.html:57]: Please remove unnecessary comments in your codebase. This makes code your unclean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <section id="feature">
+      <iframe id="video" src="https://www.youtube.com/watch?v=HIj8wU_rGIU" frameborder="0" allow="autoplay encrypted-media" allowfullscreen></iframe>
+    </section>
+    <section id="howitworks">
+     <form id="form" action="https://wwww.freecodecamp.com/email-submit"> 
+       <input id="email" name="email" type="email" placeholder="Input your email" required> 
+       <input id="submit" class="btn" type="submit" value="submit">
+
+       
+
```
Comment (Obote) [inline index.html:42]: Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" />
+     <nav id="nav-bar">
+       <ul>
+         <li><a class="nav-link" href="#feature">feature</a></li>
+         <li><a class="nav-link" href="#howitworks">How it Works</a></li>
+         <li><a class="nav-link" href="#pricing">Pricing</a></li>
+       </ul>
+
```
Comment (Obote) [inline index.html:29]: Please delete the extra blank line here

**Review by Obote — COMMENTED** [review]
Hello!
Great work and effort you have put into this project so far.

### Highlights
- Good use of Semantic HTML concepts.
- Descriptive commit messages

However, there is always room for improvement. Your long hours of work are very much appreciated. However, the code review process is for improving your code, not for criticizing it.

### Changes Required
 Please check the comments under the review.

File: index.html (hunk 1 above)
Comment (teekaytech) [reply index.html:57]: For your comment to be clear and easy to understand, you need to ensure that your comment is free of grammatical or spelling errors. I think the best way to ensure this, for now, is to read the comment once or twice after composing them.

In the future, you'll be guided on how to tools like [Grammarly](https://app.grammarly.com/) to check your grammatical construction, but you can still check it out if you're feeling ambitious :grinning: 

I suggest you reconstruct the statement above.

File: index.html (hunk 2 above)
Comment (teekaytech) [reply index.html:42]: _**[ OPTIONAL ]**_

Thank you for making such a polite request and stating the reason :+1: 
I believe you can make your review more human-friendly by using a couple of emojis and/or gifs in the review :skull: 
Emojis/gifs are here to stay and communication is better enhanced when they're used :blush:

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Obote :wave:

__This is a nice review you've made. You've used the GitHub Code Review features quite well and you've also stated the reasons why you requested the changes :handshake: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Obote 🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

You are reviewing like a pro!

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**